app/views/users/show.html.erb

```diff
@@ -28,6 +28,22 @@
 </div>
   </div>
 
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h4>Arbeitszeiten</h4>
+    </div>
+    <div class="panel-body">
+      <% @user.years_and_months_of_existence.each do |month| %>
+  	    <p>
+          <%= link_to Date::MONTHNAMES[month.last] + " " + month.first.to_s + ":  ", work_days_path(month: month.last, year: month.first)%>
+          <% current_user.projects_for_month(month.first, month.last).each do |project|%>
```

Comment:
Zudem kann ich Einträge erstellen zum Beispiel für den nächsten und letzten Monat aber die werden mir in der Übersicht nicht angezeigt...